packages/babel-plugin/src/styled/__tests__/tagged-template-expression.test.ts

```diff
@@ -908,7 +908,7 @@ describe('styled tagged template expression', () => {
         props.isPrimary
           ? \`
             background-color: \${({ isLoading }) => (isLoading ? colors.N20 : colors.N40)};
-            color: \${({ loading: l }) => (l ? colors.N50 : colors.N10)};
+            color: \${({ loading }) => (loading ? colors.N50 : colors.N10)};
```

Comment:
is this change just to make the test case easier to understand

Reply:
Reverted—didn't intend to change, was debugging!